### PR TITLE
Update autosubscribe-arn-destination.md

### PR DESCRIPTION
**Pull request by astefanski-sumo**
Broken local link fixed (#configuringparameters)

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/collect-from-other-data-sources/autosubscribe-arn-destination.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ To deploy LogGroup Lambda Connector, do the following:
 1. In the Serverless Application Repository, search for **sumologic**.
 1. Select **Show apps that create custom IAM roles or resource policies** check box.
 1. Click the **sumologic-loggroup-connector**,link, and then click **Deploy**.
-1. In the **AWS Lambda \> Functions \> Application Settings** panel, configure the parameters as described in [configuring parameters](#configuring-parameters).
+1. In the **AWS Lambda \> Functions \> Application Settings** panel, configure the parameters as described in [configuring parameters](#configuringparameters) section below.
```
Comment (stacykor) [inline docs/send-data/collect-from-other-data-sources/autosubscribe-arn-destination.md:30]:
```suggestion
1. In the **AWS Lambda \> Functions \> Application Settings** panel, configure the parameters as described in [configuring parameters](/docs/send-data/collect-from-other-data-sources/autosubscribe-arn-destination/#configuringparameters).
```

Comment (stacykor) [reply docs/send-data/collect-from-other-data-sources/autosubscribe-arn-destination.md:30]: try the full path and see if the build still rejects you.

**Review by stacykor — CHANGES_REQUESTED** [review]

**Review by kimsauce — APPROVED** [review]
looks good - anchor link worked for me

File: docs/send-data/collect-from-other-data-sources/autosubscribe-arn-destination.md (hunk 1 above)
Comment (kimsauce) [reply docs/send-data/collect-from-other-data-sources/autosubscribe-arn-destination.md:30]: Hi Stacy, the cause of the build failure was due to something in the github actions.

**Pull request merged**